### PR TITLE
Chore/update rest api definition

### PR DESCRIPTION
**Pull request by ffiebig**
Update REST API definition to last version (2.2)

### CODE REVIEW

File: referencia/webpayrest/README.md [hunk 1]
```diff
@@ -570,22 +580,154 @@ Content-Type: application/json
...
+```
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+type  <br> <i> String </i> | Tipo de reembolso (REVERSE. NULLIFY). Largo máximo: 10
+authorization_code  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
+authorization_date  <br> <i> String </i> | Fecha y hora de la autorización.
+balance  <br> <i> Decimal </i> | Saldo actualizado de la transacción (considera la venta menos el monto anulado). Largo máximo: 17
+nullified_amount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
+response_code <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0,de lo contrario el pago no fue realizado. Largo Máximo: 2
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: This should be: "Código de resultado de la Anulación/Reembolso"

File: referencia/webpayrest/README.md [hunk 2]
```diff
@@ -667,9 +809,14 @@ Content-Type: application/json
  "details": [
      {
          "amount": 10000,
-         "commerce_code": Próximamente...,
+         "commerce_code": "Próximamente...",
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: Isn't this the same commerce_code below?

File: referencia/webpayrest/README.md [hunk 3]
```diff
@@ -1053,22 +1192,25 @@ capturedAmount  <br> <i> Decimal </i> | Monto que se desea capturar. Largo má
...
-token  <br> <i> String </i> | Token de la transacción. Largo máximo: 64
-authorizationCode  <br> <i> String </i> | Código de autorización de la captura diferida. Largo máximo: 6
-authorizationDate  <br> <i> String </i> | Fecha y hora de la autorización.
-capturedAmount  <br> <i> Decimal </i> | Monto capturado. Largo máximo: 6
+type  <br> <i> String </i> | Tipo de reembolso (REVERSE. NULLIFY). Largo máximo: 10
+authorization_code  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
+authorization_date  <br> <i> String </i> | Fecha y hora de la autorización.
+balance  <br> <i> Decimal </i> | Saldo actualizado de la transacción (considera la venta menos el monto anulado). Largo máximo: 17
+nullified_amount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
+response_code <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0, de lo contrario el pago no fue realizado. Largo máximo: 2
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: again, this should say `anulación` or `reembolso`

File: referencia/webpayrest/README.md [hunk 4]
```diff
@@ -1193,23 +1337,24 @@ commerceId  <br> <i> Number </i> | (Opcional) Tienda mall que realizó la trans
...
-type  <br> <i> String </i> | Tipo de reembolso (REVERSE. NULLIFY). Largo máximo: 10
-authorizationCode  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
-authorizationDate  <br> <i> String </i> | Fecha y hora de la autorización.
-balance  <br> <i> Decimal </i> | Saldo actualizado de la transacción (considera la venta menos el monto anulado). Largo máximo: 17
-nullifiedAmount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
+token  <br> <i> String </i> | Token de la transacción. Largo máximo: 64
+authorization_code  <br> <i> String </i> | Código de autorización de la captura diferida. Largo máximo: 6
+authorization_date  <br> <i> String </i> | Fecha y hora de la autorización.
+captured_amount  <br> <i> Decimal </i> | Monto capturado. Largo máximo: 6
+response_code  <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0,de lo contrario el pago no fue realizado. Largo máximo: 2
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: `captura`?

File: referencia/webpayrest/README.md [hunk 5]
```diff
@@ -335,10 +345,10 @@ Content-Type: application/json
 
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
-buyOrder  <br> <i> String </i> | Orden de compra de la tienda. Este número debe ser único para cada transacción. Largo máximo: 26. La orden de compra puede tener: Números, letras, mayúsculas y minúsculas, y los signos <code>&#124;_=&%.,~:/?[+!@()>-</code>
-sessionId  <br> <i> String </i> | (Opcional) Identificador de sesión, uso interno de comercio, este valor es devuelto al final de la transacción. Largo máximo: 61
+buy_order  <br> <i> String </i> | Orden de compra de la tienda. Este número debe ser único para cada transacción. Largo máximo: 26. La orden de compra puede tener: Números, letras, mayúsculas y minúsculas, y los signos <code>&#124;_=&%.,~:/?[+!@()>-</code>
```
[the remark this answers is not in the document]
Comment (ffiebig) [reply referencia/webpayrest/README.md]: By this you mean the change from camel case to snake case ?

File: referencia/webpayrest/README.md (hunk 1 above)
Comment (ffiebig) [reply referencia/webpayrest/README.md]: Fixed in d09e83d

File: referencia/webpayrest/README.md (hunk 3 above)
Comment (ffiebig) [reply referencia/webpayrest/README.md]: Fixed in d09e83d

File: referencia/webpayrest/README.md (hunk 4 above)
Comment (ffiebig) [reply referencia/webpayrest/README.md]: Fixed in d09e83d

File: referencia/webpayrest/README.md (hunk 2 above)
Comment (ffiebig) [reply referencia/webpayrest/README.md]: Fixed in f40d528, commerce code can't be visible, they shouldn't be displayed yet.

File: referencia/webpayrest/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (afiebig) [reply referencia/webpayrest/README.md]: yeah!

[the remark this answers is not in the document]
Comment (ffiebig) [reply referencia/webpayrest/README.md]: This way the docs are close as it should be used, using camel case on the left and snake case on the right might cause confusion

File: referencia/webpayrest/README.md [hunk 6]
```diff
@@ -3061,11 +3222,11 @@ Content-Type: application/json
...
 type  <br> <i> String </i> | Tipo de reembolso (REVERSE. NULLIFY). Largo máximo: 10
-authorizationCode  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
-authorizationDate  <br> <i> String </i> | Fecha y hora de la autorización.
-nullifiedAmount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
+authorization_code  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
+authorization_date  <br> <i> String </i> | Fecha y hora de la autorización.
+nullified_amount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
 balance  <br> <i> Decimal </i> | Saldo actualizado de la transacción (considera la venta menos el monto anulado). Largo máximo: 17
-responseCode  <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0, de lo contrario el pago no fue realizado. Largo máximo: 2
+response_code  <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0, de lo contrario el pago no fue realizado. Largo máximo: 2
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: This is a reverse/nullify

Comment (ffiebig) [reply referencia/webpayrest/README.md]: Fixed in b0ca77f

**Pull request merged**